### ISSUE TITLE
Admin page

### ISSUE THREAD

**Issue by aranasaurus** [pdx-cocoaheads/pdxcocoaheads.com#3]
It'd be nice to have an admin page that would allow me to post a meeting description, date and location and have it post that data for me to meetup.com, calagator.org and cocoaheads.org... so I don't have to do that replication myself. Or if we could set something up that fetches the next meeting from meetup.com and then propagates it out to the other two would work too.

**Comment by aranasaurus** [comment]
This task is dependent on #6 and #8
